Tools/PIDML/simpleApplyPidOnnxModel.cxx

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file simpleApplyPidModelTask
+/// \brief A simple example for using PID obtained from the PID ML ONNX Model. See https://github.com/saganatt/PID_ML_in_O2 for more detailed instructions.
+///
+/// \author Maja Kabus <mkabus@cern.ch>
+
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "Common/Core/PID/PIDResponse.h"
+#include "Tools/PIDML/pidOnnxModel.h"
+
+#include <string>
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::framework::expressions;
+
+namespace o2::aod
+{
+namespace mlpidresult
+{
+DECLARE_SOA_INDEX_COLUMN(Track, track); //! Track index
+DECLARE_SOA_COLUMN(MlPid, mlPid, int);  //! PID predicted by the model
+} // namespace mlpidresult
+DECLARE_SOA_TABLE(MlPidResults, "AOD", "MLPIDRESULTS", o2::soa::Index<>, mlpidresult::TrackId, mlpidresult::MlPid);
+} // namespace o2::aod
+
+struct SimpleApplyOnnxModelTask {
+  PidONNXModel pidModel; // One instance per model, e.g., one per each pid to predict
```

Comment:
Otherwise we would need to load each model inside process(), and this will be slower.